data_structures/linked_list/circular_doubly_linked_list.c

```diff
@@ -0,0 +1,224 @@
+/*
+ * @file
+ *
+ * @details
+ * Circular [Doubly Linked List](https://en.wikipedia.org/wiki/Doubly_linked_list) combines the properties of a doubly linked list and a circular linked list in which two consecutive elements are linked or connected by the previous. Next, the pointer and the last node point to the first node via the next pointer, and the first node points to the last node via the previous pointer.
+ *
+ * In this implementation, the menu-driven code is written along with functions to insert at the start, insert at the last index, delete the first node, delete the last node, and display list functions are coded.
+ 
+ * @author [Sahil Kandhare](https://github.com/SahilK-027)
+ *
+ */
+
+#include <stdio.h>     /// for IO operations
+#include <stdlib.h>    /// for including functions involving memory allocation such as `malloc`
+#include <inttypes.h>  /// to provide a set of integer types with universally consistent definitions that are operating system-independent
+/**
+ * @brief Node Structure Cicular Doubly linked list.
+ */
+struct node
+{
+    struct node *prev, *next;         ///< List pointers
+    int64_t value;                   ///< Data stored on each node
```

Comment:
Should be initialized as 0 and probably as `uint64_t`. I don't think this will store negative values.
```suggestion
    uint64_t value = 0;                   ///< Data stored on each node
```